src/docs/background/23-how-capabilities-work/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: How Capabilities work?
+slug: how-capabilities-work
+updated: 2019-12-19
+category: background
+ingress:
+  Capabilities framework offers tools to manage regulatory aspects of
+  provider onboarding. This article outlines mandatory changes to the
+  framework and instructions regarding your actions on reacting to those
+  changes.
+published: true
+---
+
+## Introduction
+
+Coping with regulatory environment regarding provider onboarding can
+be a complex and daunting endeavour. Fulfilling these requirements
+means that you need to be collecting a variety of information from
+your providers in order to be able to process payments. Stripe has a
+mechanism for controlling the amount of information collected from providers called
+[Capabilities](https://stripe.com/docs/connect/capabilities-overview).
+
+Capabilities are a collection of settings that can be requested for
+each provider. What Capabilities are required determines what
+information Stripe requires to be collected from the providers.
+
+## Changes to Capabilities framework
+
+Stripe is introducing mandatory changes to the Capabilities framework.
+In the beginning of 2020, Stripe will enforce that Capabilities are
+required for all accounts, existing and new. Stripe’s deadlines for
```

Comment:
Having both "enforce" and "required" kind of repeats the same thing. How about "Stripe will enforce Capabilities for all accounts" or "Stripe will require Capabilities from all accounts" etc. ?